lib/parsers/moov_parser/decoder.rb

```diff
@@ -72,7 +72,7 @@ def find_video_trak_atom(atoms)
 
     trak_atoms.find do |trak_atom|
       hdlr_atom = find_first_atom_by_path([trak_atom], 'trak', 'mdia', 'hdlr')
```

Comment:
This means `hdlr_atom` is `nil`. I haven't checked the format specifications, but would that be a valid scenario? 🤔